src/tests/ftest/container/simple_create_delete_test.py

```diff
@@ -51,48 +52,32 @@ def test_container_basics(self):
         Test basic container create/destroy/open/close/query.  Nothing fancy
         just making sure they work at a rudimentary level
 
-        :avocado: tags=container,containercreate,containerdestroy,basecont
+        :avocado: tags=pr,container,containercreate,containerdestroy,basecont
         """
-
-        pool = None
-        hostlist = None
-
         try:
-            hostlist = self.params.get("test_machines", '/run/hosts/*')
-            hostfile = write_host_file.write_host_file(hostlist,
-                                                       self.workdir)
+            self.hostlist_servers = self.params.get("test_machines", '/run/hosts/*')
```

Comment:
Does this yield the single host list (`test_machines1`) or the list of 6 servers (`test_machines6`) from the yaml file.  I do not see a reference to either path in the original code (either `/run/hosts/test_amchines1/test_machines` or `/run/hosts/test_amchines6/test_machines`).  If only one server is required can we cleanup the yaml file?

Reply:
The yaml is using a !mux on the 'hosts' tag. So it should have a pass with one machine and a pass with 6 machines. I am not sure what the purpose of have 6 machines is, we'd have to verify the intent with the original author. Since this test is for basic container (create, destroy, open, query) functions, I have no issue cleaning up the yaml to only have one host.